CameraOverlay/overlay_all_stats.py

```diff
@@ -129,7 +131,18 @@ def on_message(self, client, userdata, msg):
 				self.data["reed_velocity"] = 0
 				self.data["reed_distance"] = 0
 				self.data["count"] = 0
-
+		
+		elif topic == str(topics.DAShboard.recieve_message):
+			message = msg.payload.decode("utf-8")
+
+			# reset counter once message is recieved 
+			self.frame_counter = 0  
+
+			# Display Message
+			self.message_canvas.draw_text("{0}".format(message), (190, self.text_height * 5), size= 1, colour=Colour.red)
+		
+		if self.frame_counter >= 70:
```

Comment:
`70` is a magic number here - what does it mean? Consider making a constant `frames_per_second` or something like that, then to wait for example 3 seconds, we can use `3 * self.frames_per_second`

Reply:
Could even bring that out into another constant, `message_display_time` or something